templates/zerver/api/changelog.md

```diff
@@ -11,6 +11,14 @@ below features are supported.
 
 ## Changes in Zulip 5.0
 
+**Feature level 72**
+
+* [`GET /events`](/api/get-events): Renamed `max_logo_file_size` to
+  `max_logo_file_size_mib`.
```

Comment:
From reading the code, I think only `realm_upload_quota_mib` appears in `get_events` responses.  

(I suspect we have a documentation bug that `max_logo_file_size` was incorrectly listed in the `GET /events` documentation, and should be removed -- our OpenAPI tests do not have an easy way to check for fields that don't actually exist but appear in the documentation; only the other way around).

In general, you should be auditing for this sort of bug when doing a `git grep` for these identifiers as part of editing documentation, since errors in this sort of thing are quite problematic.